projects/1.20/assets/the-outer-end/outer_end/lang/zh_cn.json

```
@@ -0,0 +1,215 @@
+{
+  "block.outer_end.azure_stem": "碧蓝木原木",
+  "block.outer_end.azure_pith": "碧蓝木木髓",
+  "block.outer_end.azure_stripped_stem": "去皮碧蓝木原木",
+  "block.outer_end.azure_stripped_pith": "去皮碧蓝木木髓",
+  "block.outer_end.azure_planks": "碧蓝木木板",
+  "block.outer_end.azure_leaves": "碧蓝木树叶",
+  "block.outer_end.azure_stamen": "碧蓝木花蕊",
+  "block.outer_end.azure_bud": "碧蓝木树苗",
+  "block.outer_end.azure_grass": "碧蓝草方块",
+  "block.outer_end.ender_roots": "末影根茎",
+  "block.outer_end.tall_ender_roots": "高末影根茎",
+  "block.outer_end.azure_sprouts": "碧蓝苗",
+  "block.outer_end.azure_vines": "碧蓝藤蔓",
+  "block.outer_end.azure_fence": "碧蓝木栅栏",
+  "block.outer_end.azure_fence_gate": "碧蓝木栅栏门",
+  "block.outer_end.azure_door": "碧蓝木门",
+  "block.outer_end.azure_trapdoor": "碧蓝木活板门",
+  "block.outer_end.azure_pressure_plate": "碧蓝木压力板",
+  "block.outer_end.azure_button": "碧蓝木按钮",
+  "block.outer_end.azure_slab": "碧蓝木台阶",
+  "block.outer_end.azure_stairs": "碧蓝木楼梯",
+  "block.outer_end.azure_sign": "碧蓝木告示牌",
+  "block.outer_end.azure_hanging_sign": "碧蓝木悬挂式告示牌",
+  "block.outer_end.azure_berry_vine": "碧蓝浆果藤",
+  "block.outer_end.azure_berry_vine_top": "碧蓝浆果藤顶端",
+  "block.outer_end.potted_azure_bud": "碧蓝木盆栽",
+  "block.outer_end.potted_ender_roots": "末地根茎盆栽",
+  "block.outer_end.potted_azure_sprouts": "碧蓝苗盆栽",
+
+  "block.outer_end.azure_vertical_slab": "竖向碧蓝木台阶",
+  "block.outer_end.azure_vertical_planks": "竖纹碧蓝木木板",
+  "block.outer_end.azure_ladder": "碧蓝木梯子",
+  "block.outer_end.azure_bookshelf": "碧蓝木书架",
+  "block.outer_end.azure_beehive": "碧蓝木蜂窝",
```

```suggestion
  "block.outer_end.azure_beehive": "碧蓝木蜂箱",
```
建议检查是否是Buzzier Bees的兼容